frontend/views/reports.js

```diff
@@ -11,23 +13,69 @@ var Backbone = require('backbone'),
 var ReportTableView = Backbone.View.extend({
 
     initialize: function (options) {
+        this.chartEl = $(options.chartEl);
         this.report = options.report;
         this.results = null;
+        this.flotOptions = {
+            series: {stack: true,
+                     bars: {show: true,
+                            barWidth: 0.6,
+                            align: "center"}},
+            grid: {hoverable: true},
+            xaxis: {mode: "categories"},
+            yaxis: {min: 0,
+                    tickDecimals: 0}};
+        $("<div id='tooltip'></div>").appendTo("body");
```

Comment:
I was trying to put it in the handlebar template. Putting it in index.html works perfectly, thanks!